### ISSUE TITLE
identical_contents: false positive with chaining

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1428]
```abap
  IF p_obje = abap_true.
    WRITE: / 'Found', lv_lines, 'dependencies for', p_type, p_name.
  ELSE.
    WRITE: / 'Found', lv_lines, 'dependencies for the following packages:'.
    LOOP AT lt_packages INTO lv_package.
      WRITE: AT /5 lv_package.
    ENDLOOP.
  ENDIF.
```

![image](https://user-images.githubusercontent.com/5888506/95655399-48dc3600-0b07-11eb-8f4c-987b5fac6243.png)

**Comment by larshp** [comment]
done, closing
